server/routes/api-v1/userRoutes.ts

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import * as authController from '../../controllers/authController';
+
+const userRouter = express.Router();
+
+userRouter.get('/login', authController.login);
+
+userRouter.get('/logged', authController.getToken);
```

Comment:
What is `logged` endpoint supposed to do? 
If it supposed to check if user is logged, it's useless, user will see `401 Unauthorized` http status when he is unauthorized/logged out

Reply:
It's part of OAuth codeflow